### ISSUE TITLE
Hi, errors when training train_RRI.sh

### ISSUE THREAD

**Issue by cuih0001** [syfafterzy/SVDNet-for-Pedestrian-Retrieval#11]
I have trained the basemodel successfully. However, when I train_RRI.sh, I got so many errors:

![image](https://user-images.githubusercontent.com/38590739/39086793-66e7cfb0-45ca-11e8-8004-24100330a26e.png)

![image](https://user-images.githubusercontent.com/38590739/39086804-80b177f2-45ca-11e8-8142-7046049af79f.png)

![image](https://user-images.githubusercontent.com/38590739/39086814-a0d0be1c-45ca-11e8-874d-04c756209faf.png)

![image](https://user-images.githubusercontent.com/38590739/39086820-c1974580-45ca-11e8-81c9-ba4d48295a26.png)

The lmdb data is fine, since I have trained the basemodel. I really do not know what is wrong.
It bothers me for a long time.
Hope you can help me. Thanks.

**Comment by ghost** [comment]
@cuih0001  I think this problem is caused by path, and you can try to write absolute path.

Check you lmdb path again. 

Good luck to you!

**Comment by cuih0001** [comment]
@Simon4john Thanks for your reply. I have changed to absolute path but it did not work. I guess it caused by calling Matlab in the for loop. But I have compiled matcaffe before I run train_RRI.sh. Really do not know what to do next?

**Comment by ghost** [comment]
Ops! I see your error message, and I agree with you that the error lies in matcaffe. I suggest you recompile matcaffe interface again, and I suggest using official caffe.

Good luck!

**Comment by cuih0001** [comment]
@Simon4john Yes, I recompiled matcaffe and also use official caffe, but still cannot solve this. 
Anyone know this issue and can give some suggestions. Thanks a lot.
